tls_decryption.md

```diff
@@ -104,7 +104,7 @@ If you see the above error "*client does not trust proxy's certificate*" while d
 
 #### Reverse engineering APKs and other advanced topics
 
-- If none of the above mentioned method works, then the app may use custom pinning logic, in which case you will need to decompile the app, reverse engineer the app, and patch it. Commonly used softwares for decompiling apps are [JADX](https://github.com/skylot/jadx), [apktool](https://apktool.org/). For a guide on how to use this, refer to [blog](https://hackernoon.com/apk-decompilation-a-beginners-guide-for-reverse-engineers) and to this [video tutorial](https://www.youtube.com/watch?v=QwwLSyRzNwo) by John Hammond
+- If none of the above mentioned method works, then the app may use custom pinning logic, in which case you will need to decompile the app, analyse the code and then patch and rebuild it. Commonly used softwares for decompiling and patching apps are [JADX](https://github.com/skylot/jadx), [apktool](https://apktool.org/) or the [APKLab](https://github.com/APKLab/APKLab) extension in Visual studio code.Refer to this [blog](https://braincoke.fr/blog/2021/03/android-reverse-engineering-for-beginners-decompiling-and-patching/)  for a full guide on reverse-engineering and decompiling app using above tools. 
```

Comment:
Note: space missing after full stop. Also duplicated space before "for a full guide on"